pointers/pointers.c

```diff
@@ -21,7 +24,13 @@ void swap(int* a, int* b)
 */
 char *find_char(char *str, int c)
 {
-
+    while(*str){
```

Comment:
make sure your pointers will be pointing to your memory address properly, debug, and test it out :)